store/postgres/src/relational_queries.rs

```diff
@@ -213,6 +219,24 @@ impl<'a> QueryFragment<Pg> for QueryValue<'a> {
                         out.push_sql("[]");
                         Ok(())
                     }
+                    // TSVector will only be in a Value::List() for inserts so "to_tsvector" can always be used here
+                    ColumnType::TSVector((language, _)) => {
+                        let mut iter = values.iter().peekable();
+                        out.push_sql("(");
+                        while let Some(value) = iter.next() {
+                            out.push_sql("to_tsvector(");
+                            out.push_bind_param::<Text, _>(&language.as_sql())?;
```

Comment:
Since we know that `language` is a safe string, you could just write it out directly without going through a bind variable.